client/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "client",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "axios": "^0.18.0",
+    "passport": "^0.4.0",
```

Comment:
Do we need `Passport` module for this project?

Reply:
I think Passport will be what we use if we get to authentication. However, since authentication is a bonus feature I'm fine with this getting stripped out for now